inc/commonglpi.class.php

```diff
@@ -842,8 +842,6 @@ function showTabsContent($options = []) {
             unset($cleaned_options['stock_image']);
          }
          if ($this->getType() == 'Ticket') {
-            $this->input = $cleaned_options;
-            $this->saveInput();
```

Comment:
This is a "hack", and removing it causes description field content emptyness when form is reloaded (when category is changed, for instance).

Reply:
Ok, I propose to use this code only when creating a ticket
done

Reply:
`$this->input = $cleaned_options;` is used later to restore uploaded files (`$uploads['_filename'] = $this->input['_filename'];`). 

On update, as `$this->input` is not set, files uploaded before form reload are not restored and so images are stored as base64 in rich text.

I guess the solution would be to set `$this->input` on `restoreSavedValues` method.